websites/C/CHZZK/metadata.json

```diff
@@ -5,12 +5,21 @@
 		"id": "374905512661221377",
 		"name": "Slowlife"
 	},
+	"contributors": [
+		{
+			"name": "rai_ny._.",
+			"id": "741973166364164099"
+		}
+	],
```

Comment:
Too small of a change imo for Contributor.

Reply:
OK I remove it